tests/test_enterprise/api/test_views.py

```diff
@@ -642,7 +650,7 @@ def test_api_views(self, factory, url, sorting_key, model_items, expected_json):
     @ddt.unpack
     def test_enterprise_customer_catalogs_list(self, is_staff, is_linked_to_enterprise):
         """
-        ``enterprise-catalogs``'s list endpoint should serialize the ``EnterpriseCustomerCatalog`` model.
+        ``enterprise_catalogs``'s list endpoint should serialize the ``EnterpriseCustomerCatalog`` model.
```

Comment:
I know it's just a comment -- but do you want to revert this too as per https://github.com/edx/edx-enterprise/pull/202#discussion_r142038744?

Reply:
Nevermind!